testing/dockerfiles/baz/Dockerfile.tmpl

```diff
@@ -0,0 +1,5 @@
+FROM    ubuntu
+RUN     apt-get update && apt-get install -y wget
+RUN     wget -q -O /dummy.deb ${IP_ADDRESS}/dummy.deb || true
+RUN     dpkg --install dummy.deb || true
```

Comment:
shouldn't `|| true` here -- this should always succeed (or your package is broken)

Reply:
The package being broken sometimes is intentional. It'll build an image and download/install the dummy.deb if the nginx container serving it is up. Otherwise it'll build an image without it by attempting and failing these steps, but completing the build anyway.

It was the simplest way I could come up with to create two images with the same set of Dockerfile commands but have a different `dpkg -l` result at build time.

Reply:
let's add a comment saying essentially this, it's v non obvious otherwise :) (neat idea though, I was wondering how this was going to get tested)